.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,37 @@
+name: CIFuzz
+on: [pull_request]
+permissions:
+  contents: read
+jobs:
+ Fuzzing:
+   runs-on: ubuntu-latest
+   permissions:
+     security-events: write
+   steps:
+   - name: Build Fuzzers
+     id: build
+     uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@a790ab47e189e5e3b4941b991f4784ec769a9e70
```

Comment:
Pinning is a really bad idea with CIFuzz.

Reply:
@floitsch let me know what you think -- I pinned due to your request https://github.com/google/double-conversion/pull/203#discussion_r1260135811 but given the feedback from @jonathanmetzman let me know which you prefer. There aren't many updates in https://github.com/google/oss-fuzz/tree/master/projects/double-conversion which may speak for keeping the hashes.

Reply:
There are almont no updates for double-conversion's oss-fuzz. I think we can pin the version.

The hash-pinning was recommended by @joycebrum.

I can get convinced either way: given the limited permissions there isn't a lot of risk (not that I think that the oss-fuzz repository would get taken over), but with the limited amount of changes pinning should also be fine.

Reply:
So actually I was wrong, even updates to the integration are not affected by pinning.
The only thing that actually gets pinned are these files: https://github.com/google/oss-fuzz/blob/master/infra/cifuzz/actions/build_fuzzers/action.yml https://github.com/google/oss-fuzz/blob/master/infra/build_fuzzers.Dockerfile
That's because once build_fuzzers.Dockerfile gets built, it pulls from gcr.io/oss-fuzz-base/cifuzz-base See [here](https://github.com/google/oss-fuzz/blob/master/infra/build_fuzzers.Dockerfile#L19)) which is continuously updated.

I don't think pinning is likely to break much here. But it's also not providing any security. I think some breakages outweighs 0 security, but either way it isn't too important.
I don't update these files too often.